src/GoTrueClient.ts

```diff
@@ -37,6 +37,14 @@ import type {
 
 polyfillGlobalThis() // Make "globalThis" available
 
+type CallRefreshTokenResult = {
```

Comment:
This type is solely used in private methods and properties. Not sure if it nevertheless should go to `lib/types.ts`? Both is fine for me.

Reply:
Yeah let's put this in `lib/types.ts` to keep things consistent 😄